client/src/main.rs

```diff
@@ -546,13 +550,18 @@ fn main() {
                             OpaqueCall::from_tuple(&propose_add_location_batch)
                         };
                     }
-                    // ---- send xt's to chain
-                    if api.get_current_phase().unwrap() != CeremonyPhaseType::Registering {
-                        error!("Wrong ceremony phase for registering new locations for {}", cid);
-                        error!("Aborting without registering additional locations");
-                        std::process::exit(exit_code::WRONG_PHASE);
+
+                    if matches.dryrun_flag() {
+                        println!("0x{}", hex::encode(add_location_batch_call.encode()));
+                    } else {
```

Comment:
From what I can see in the diff, it does not make sense to check for the dry_run flag twice. I'd prefer that the branching occurs only once.

Reply:
As discussed, I'll leave it because we need a more fundamental cli refactoring to solve this nicely